spec/views/users/phone_confirmation/show.html.slim_spec.rb

```diff
@@ -12,4 +12,39 @@
 
     expect(rendered).to have_content('A confirmation code has been sent to 12345')
   end
+
+  it 'has a localized title' do
+    expect(view).to receive(:title).with(t('titles.enter_2fa_code'))
+
+    render
+  end
+
+  it 'asks the user to confirm thier phone number' do
+    render
+
+    expect(rendered).to have_content(t('forms.phone_confirmation.header_text'))
+  end
+
+  it 'allows user to resend code' do
+    render
+
+    expect(rendered).to have_link('request it be sent again', href: phone_confirmation_send_path)
+  end
+
+  it 'allows user to re-enter phone number' do
+    @reenter_phone_number_path = 'reenter_url'
+    render
+
+    expect(rendered).to have_link('entering it again', href: 'reenter_url')
+  end
```

Comment:
Since there is logic around what the `@reenter_phone_number_path` is set to, we need to test that logic here. Otherwise, this test will always pass, even if we change the controller code. We want to make sure that when the user is setting up their phone for the first time, the URL is different from when they are updating their existing phone.